### ISSUE TITLE
Login attempt times out while Google calls to provide the security code

### ISSUE THREAD

**Issue by larstiq** [rogora/hangish#49]
On starting Hangish there is a prompt for username and password. On filling that and pressing login, Hangish briefly shows a PIN prompt.  Then Google's robot caller calls to tell me what my security code is and by the time the conversation is done Hangish has reverted to username/password prompt. Rinse, lather, repeat.

Since the security code is the same in a short period, my workaround was to decline the call, enter the security code in the PIN prompt and logging in with that.  (Hangish was displaying something about auth cookie failed or whatnot, went unresponsive for a bit, but finally did log in).

**Comment by jovirkku** [comment]
I have the 2-phase Google sign in active (https://myaccount.google.com/signinoptions/two-step-verification). It means that whenever i sign in to a Google service, they will send me a security code to my primary phone (alternatively i can get a confirmation dialog without any code to that phone). In case of trying to sign in to Hangish I am not getting any code or a confirmation dialog. I am not able to get in to Hangish at all. I have also created the Google spare codes in their service but they do not seem to let me in, either: Hangish keeps showing a spinner and the Wait button persists at the top right corener of the display.

**Comment by rogora** [comment]
Thank you for the report, larstiq. I typically receive a sms with the code, so I never noticed this issue.
Thanks for sharing your workaround, I'll see if I can fix hangish.

**Comment by rogora** [comment]
jovirkku, I cannot reproduce your problem. Hangish works correctly with the 2 step verification. You should receive your code an be able to input that in the corresponding form, shown by hangish.
You should use your main credentials, not the application codes that you can create with the Google services. 
Also, notice that the authentication may take a while (maybe 10/15 seconds, sometimes).
If you're sure that you're doing everything correctly, you should send me debug logs so that I can check what happens.

**Comment by stephgosling** [comment]
Just another +1 for this issue. Am able to log in using either primary account user/pass, which causes the OTP to be SMSed but then it sits in the 'wait' spinner forever. It does create an oauth2.json file and if I kill hangish it then sits in the 'logging in' spinner forever.

**Comment by abranson** [comment]
As of this morning my previously logged in Hangish is spinning on 'Logging in' forever, printing the same 'SIDCC' messages to the console.

**Comment by rogora** [comment]
Thanks for the report, this is a different problem, though. 
Apparently, Google introduced a new authentication cookie, SIDCC, that is updated frequently, and hangish doesn't manage it correctly. 
Fixing the "logging in" part is easy, I've already done that, but handling correctly this new cookie is not.

**Comment by iotamudelta** [comment]
Same issue here as @stephgosling, same debug output repeating when started from terminal.

**Comment by rogora** [comment]
Yes, should be fixed, but I'm still testing the new version. Please, feel free to try it and give any feedback.
